vspec/model/vsstree.py

```diff
@@ -68,6 +68,7 @@ class VSSNode(Node):
         "$file_name$",
         "fka",
         "delete",
+        "constUID",
```

Comment:
We should discuss how integrated we want the id-attributes to be in vss-tools. By adding `constUID` (and previously `fka`) they will be considered as core attributes and accepted in all tools.

There are two alternatives, one is to consider them as extended attributes and give warning unless they are specified by `-e`, the other is to refactor vss-tools so that each tool can add their own core attributes. Or alternatively that we consider them as "global core attributes", but then we should possibly mention them in https://github.com/COVESA/vss-tools/blob/master/docs/vspec2x_arch.md

Reply:
@erikbosch can you discuss this during the public meeting today? I have been sick yesterday / today, but can finish this tomorrow if you decide on how you want it to work. I would appreciate it, thank you!!